src/kit/Toast.tsx

```diff
@@ -64,11 +83,61 @@ export const makeToast = ({
     ) : undefined,
     duration,
     message: (
-      <div className={css.message}>
-        <Icon decorative name={getIconName(severity)} />
-        {title}
-      </div>
+      <ToastThemeProvider>
+        <div className={css.message}>
+          <Icon decorative name={getIconName(severity)} />
+          {title}
+        </div>
+      </ToastThemeProvider>
     ),
   };
   notification.open(args);
 };
+
+export const useToast = (): ({ openToast: (args: ToastArgs) => void }) => {
+  const {
+    themeSettings: { theme, themeIsDark },
+  } = useTheme();
+
+  const openToast = ({
+    title,
+    severity = 'Info',
+    closeable = true,
+    duration = 4.5,
+    description,
+    link,
+  }: ToastArgs) => {
+    const args = {
+      closeIcon: closeable ? (
+        <UIProvider theme={theme} themeIsDark={themeIsDark}>
+          <Icon decorative name="close" />
+        </UIProvider>
+      ) : null,
+      description: description ? (
+        link ? (
+          <UIProvider theme={theme} themeIsDark={themeIsDark}>
+            <div>
+              <p>{description}</p>
+              {link}
+            </div>
+          </UIProvider>
+        ) : (
+          description
+        )
+      ) : undefined,
+      duration,
+      message: (
+        <UIProvider theme={theme} themeIsDark={themeIsDark}>
+          <ToastThemeProvider>
+            <div className={css.message}>
+              <Icon decorative name={getIconName(severity)} />
+              {title}
+            </div>
+          </ToastThemeProvider>
+        </UIProvider>
+      ),
+    };
+    notification.open(args);
```

Comment:
doesn't `notification.open` take `classname`? i'm seeing it here in https://ant.design/components/notification